bouncer/tests/btc_vault_swap.ts

```diff
@@ -22,7 +22,7 @@ const commissionBps = 100;
 
 interface VaultSwapDetails {
   chain: string;
-  nulldata_utxo: string;
+  nulldata_payload: string;
```

Comment:
same

Reply:
This one has to remain like this because its an interface for a rust struct.